test/run-in-travis.sh

```diff
@@ -21,8 +21,26 @@ regression_tests() {
 	sh /usr/share/crmsh/tests/regression.sh
 }
 
-unit_tests
-rc_unittest=$?
-configure
-make_install
-regression_tests && exit $rc_unittest
+bootstrap_tests() {
+        SUFFIX="${1:-*}"
+	echo "** Bootstrap process tests using python-behave"
+        behave --no-logcapture --tags "@bootstrap" --tags "~@wip" /usr/share/crmsh/tests/features/bootstrap_$SUFFIX.feature
+}
+
+case "$1" in
+	build)
+		configure
+		make_install
+		exit $?;;
+	bootstrap)
```

Comment:
I wouldn't use here this `bootstrap` entry. I would name it `functional_tests` or similar. If in the future we decide to add additional test scenario it will make more sense

Reply:
changed